incubating/octopusdeploy-deploy-release-tenanted/deploy-release-tenanted.yml

```diff
@@ -0,0 +1,174 @@
+version: "1.0"
```

Comment:
Rename the file to step.yaml. It the special name that that our syncing service looking for in the step folder.